src/db/store.rs

```diff
@@ -44,6 +44,12 @@ impl SqliteStore {
         self::last_inserted_row_id(&conn)
     }
 
+    pub fn load_users(&self) -> DbResult<Vec<User>> {
+        let conn = self.0.get()?;
+        let users = users::table.load(&conn)?;
+        Ok(users)
+    }
+
```

Comment:
We might want this in the future, but for now let's just make an `exists` check:

```suggestion
    pub fn users_exist(&self) -> DbResult<bool> {
        let conn = self.0.get()?;
        let r = diesel::select(diesel::dsl::exists(users::table.select(users::all_columns)))
            .get_result(&conn)?;
        Ok(r)
    }
```